cmake/f3dPlugin.cmake

```diff
@@ -276,6 +279,23 @@ macro(f3d_plugin_build)
       LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT plugin)
   endif()
 
+
+  # Install configurations folders
+  if(DEFINED f3d_INCLUDE_DIR)
```

Comment:
this condition is already tested above, shouldn't we factorize the code a bit?

Reply:
good point